src/templates/BlogPost.js

```diff
@@ -59,7 +59,11 @@ const BlogPostTemplate = ({ data, location }) => {
         <div className={classes.callToActionWrapper}>
           <InterstitialTitle text="Become a Backstage expert" />
           <p className={classes.callToActionParagraph}>
-            Get the latest Backstage news in your inbox each week.
+            To get the latest news, deep dives into Backstage features, and a roundup of recent
+            open-source action, sign up for Roadie&apos;s Backstage Weekly.{' '}
```

Comment:
out of curiosity, what is `{' '}` for?

Reply:
It's to add a whitespace between the `Weekly.` and the `<a href...`. If I remove it, HTML collapses the whitespace and the text of the anchor tag pushes up against the period.

The other option I know is to add a `&nbsp;` but that will refuse to line break correctly so AFAIK this solution is the best way to do it.